.github/workflows/build.yml

```diff
@@ -20,9 +20,9 @@ jobs:
         include:
           - os: buildjet-4vcpu-ubuntu-2204-arm
             arch: arm64
-          - os: ubuntu-20.04
+          - os: ubuntu-22.04
```

Comment:
How do we know that this unrelated change to the amd64 build doesn't break anything?